Goobi/src/messages/messages_de.properties

```diff
@@ -600,8 +600,9 @@ importplugin=Plugin f\u00FCr den Import
-impressumTextDemo=Die nachstehenden Informationen enthalten die gesetzlich vorgesehenen Pflichtangaben zur Anbieterkennzeichnung, sowie wichtige rechtliche Hinweise zur Internetpr\u00E4senz. <br/><br/><b>Anbieter</b><br/>Anbieter dieser Internetpr\u00E4senz ist im Rechtssinne die Nieders\u00E4chsische Staats- und Universit\u00E4tsbibliothek G\u00F6ttingen. <br/><br/><b>Rechtliche Hinweise zum Urheberrecht </b><br/>Das Layout der Homepage, die verwendeten Grafiken sowie die sonstigen Inhalte sind urheberrechtlich gesch\u00FCtzt. <br/>Goobi hat auf seinen Seiten Links zu anderen Seiten im Internet gelegt. Trotz sorgf\u00E4ltiger inhaltlicher Kontrolle \u00FCbernehmen wir keine Haftung f\u00FCr die Inhalte der externen Links, auf die wir hinweisen. Diese Erkl\u00E4rung gilt f\u00FCr alle auf der Website angezeigten Links. <br/>Goobi ist bem\u00FCht, sein Webangebot stets aktuell und inhaltlich richtig anzubieten. Es ist jedoch nicht m\u00F6glich, das Auftreten von Fehlern v\u00F6llig auszuschlie\u00DFen. Wir \u00FCbernehmen deswegen keine Gewahr f\u00FCr die Vollst\u00E4ndigkeit oder Aktualit\u00E4t der in unserem Webangebot eingestellten Informationen. Dies bezieht sich auf eventuelle Sch\u00E4den materieller oder ideeller Art Dritter, die durch die Nutzung dieser www-Seiten verursacht sein k\u00F6nnten.
+impressumTextSUBGoettingen=Die nachstehenden Informationen enthalten die gesetzlich vorgesehenen Pflichtangaben zur Anbieterkennzeichnung, sowie wichtige rechtliche Hinweise zur Internetpr\u00E4senz. <br/><br/><b>Anbieter</b><br/>Anbieter dieser Internetpr\u00E4senz ist im Rechtssinne die Nieders\u00E4chsische Staats- und Universit\u00E4tsbibliothek G\u00F6ttingen. <br/><br/><b>Rechtliche Hinweise zum Urheberrecht </b><br/>Das Layout der Homepage, die verwendeten Grafiken sowie die sonstigen Inhalte sind urheberrechtlich gesch\u00FCtzt. <br/>Goobi hat auf seinen Seiten Links zu anderen Seiten im Internet gelegt. Trotz sorgf\u00E4ltiger inhaltlicher Kontrolle \u00FCbernehmen wir keine Haftung f\u00FCr die Inhalte der externen Links, auf die wir hinweisen. Diese Erkl\u00E4rung gilt f\u00FCr alle auf der Website angezeigten Links. <br/>Goobi ist bem\u00FCht, sein Webangebot stets aktuell und inhaltlich richtig anzubieten. Es ist jedoch nicht m\u00F6glich, das Auftreten von Fehlern v\u00F6llig auszuschlie\u00DFen. Wir \u00FCbernehmen deswegen keine Gewahr f\u00FCr die Vollst\u00E4ndigkeit oder Aktualit\u00E4t der in unserem Webangebot eingestellten Informationen. Dies bezieht sich auf eventuelle Sch\u00E4den materieller oder ideeller Art Dritter, die durch die Nutzung dieser www-Seiten verursacht sein k\u00F6nnten.
 inAuswahllisteAnzeigen=In Auswahlliste anzeigen
```

Comment:
For preserving text we have a git based history, so anybody how need this could look into the history of this file. Removing this entries would result in an empty page which should avoided. Instead of this a notice should be added to installation guide to add a institute based legal notice in a custom translation message file.